packages/kotti-ui/source/kotti-comment/components/CommentHeader.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<div class="kt-comment__comment-header">
+		<div class="kt-comment__comment-header__left-wrapper">
+			<span class="kt-comment__comment-header__name" v-text="userName" />
+			<span
+				v-if="isInternalThread"
+				class="kt-comment__comment-header__internal-thread"
+			>
+				<i class="yoco" v-text="'permission'" />
+				{{ translations.internalThreadLabel }}
+			</span>
+		</div>
+		<div class="kt-comment__comment-header__right-wrapper">
+			<span
+				v-if="isModified"
+				class="kt-comment__comment-header__modified"
+				v-text="translations.editedLabel"
+			/>
+			<span
+				class="kt-comment__comment-header__creation-date"
+				v-text="createdTime"
+			/>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api'
+
+import { useTranslationNamespace } from '../../kotti-i18n/hooks'
+import { makeProps } from '../../make-props'
+import { KottiComment } from '../types'
+
+export default defineComponent({
+	name: 'CommentHeader',
+	props: makeProps(KottiComment.CommentHeader.schema),
+	setup() {
+		const translations = useTranslationNamespace('KtComment')
+
+		return {
+			translations,
+		}
+	},
+})
+</script>
+
+<style lang="scss" scoped>
+.kt-comment__comment-header {
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
```

Comment:
do you know why you needed this?

if the parent container is a flex, the children nodes should fit right in